src/app/app.routing.ts

```diff
@@ -7,6 +7,11 @@ import { LoginComponent } from './authentication/login.component';
 import { ForgotPasswordComponent } from './authentication/forgot-password.component';
 import { ResetPasswordComponent } from './authentication/reset-password.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
+import { SplashscreenComponent } from './splashscreen/splashscreen.component';
```

Comment:
not sure how ts/angular2+ works, but is `SplashscreenComponent` required in this page? it seems like an unused variable to me in this particular page, so i don't know why you are including it.

it's fine to include packages if you are using it, otherwise that's just extra payload